ykbuild/build.rs

```diff
@@ -136,8 +136,13 @@ fn main() {
         .current_dir(build_dir.as_os_str().to_str().unwrap());
 
     let mut build_cmd = Command::new("cmake");
+    let mut build_args = vec!["--build".into(), ".".into()];
```

Comment:
This is an optional nitpick, but more args use `into()` than don't, which would suggest the array would be better typed `&str` than `String`, then many of the `into()`s will go away.

Reply:
We can't do `&str` in the array because `env::var` returns a `String`.

Reply:
So borrow it :)

Reply:
In this case it's not in scope long enough to do it.

Reply:
Fair enough.